### DISCUSSION TITLE
Change wifi

### DISCUSSION THREAD

**Discussion by TheLion** [jomjol/AI-on-the-edge-device#2717] [General]
### Issue

Hi,

I am currently in the process of changing my wifi SSID and Password for my IoT devices. But I can't find the option I the AI-on-the-edge webgui to change this. 

I did read about the wlan.ini, but can't find what syntax to use, if that is the route I should take. 

Hope someone can point me in the right direction.

**Comment by friedpa** [comment]
You have to edit the wlan.ini directly, there is no WEBGUI functionality for it for security reason. You can also read that in the WIKI
Syntax:
ssid = "xxxxxxx"
password = "yyyyyyy"
hostname = "zzzzzzz"

**Reply by blauzaaan** [reply]
Hi there,
not strictly related to this, but I figured that in the web interface setup process, if your WiFi/WLAN password includes an ampersand (&), it will get truncated and not fully written to wlan.ini. It needs to be entered directly into the wlan.ini.
